src/main/java/org/jabref/gui/maintable/BibEntryTableViewModel.java

```diff
@@ -53,6 +57,30 @@ public BibEntryTableViewModel(BibEntry entry, BibDatabaseContext bibDatabaseCont
         this.linkedIdentifiers = createLinkedIdentifiersBinding(entry);
         this.matchedGroups = createMatchedGroupsBinding(bibDatabaseContext, entry);
         this.bibDatabaseContext = bibDatabaseContext;
+        this.bibPreview = createBibPreviewTooltip(entry);
+    }
+
+    private String createBibPreviewTooltip(BibEntry entry) {
```

Comment:
This is the wrong appraoch. As stated in the issuse you should use the existing methods for generating previews
See how it's done in the PrevieViewer